sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj

```diff
@@ -23,4 +23,8 @@
     <PackageReference Include="System.Text.Json" />
     <PackageReference Include="System.Threading.Tasks.Extensions" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Now that AssemblyInfo.cs was deleted, we don't need this in the csproj.
Whenever we write tests for this API we will need both again though...

Reply:
Ug, missed it. VS adds this automatically :(